src/posix/platform/resolver.cpp

```diff
@@ -97,12 +99,18 @@ void Resolver::LoadDnsServerListFromConf(void)
     {
         if (line.find(kNameserverItem, 0) == 0)
```

Comment:
nit: you can invert the condition so that the following code block can indent less.